Automattic/jetpack
# VideoPress: reload preview on oembed fail

Fixes Automattic/greenhouse#936

#### Changes proposed in this Pull Request:

Regenerate oembed preview on VideoPress oembed failure.
This fixes the problem with the video embed width/height not being known before generating the oembed preview. 

#### Does this pull request change what data or activity we track or use?
No

#### Testing instructions:
* On a sandboxed wpcom and Jetpack with VideoPress available
* Apply wpcom patch D70588-code
* Create a new post on your Jetpack
* Add a /video block and upload a portrait video
* ✅ The video block should generate the embed with the right aspect ratio (you can check out the greenhouse issue to see screenshots of previous failure)

Note: Sometimes, it works on the first try, so you may have to try multiple times until you see a core/video being displayed instead of a VideoPress video and you should see the block being reloaded at a rate between 2 to 10s. depending on the try count.
